src/components/Property/index.js

```diff
@@ -1,294 +1,422 @@
-                      <li><strong>Pincode:</strong> 380059</li>
-                    </ul>
-                  </div>
-                  <div class="level-right">
-                    <ul>
-                      <li><strong>City:</strong> {property.city}</li>
-                      <li><strong>Property Owner:</strong> Chandulal</li>
-                      <li><strong>Contact:</strong>{property.ownerPhoneNumber}</li>
-                    </ul>
-                  </div>
+		);
+    };
+
+    RenderButton = () => {
+        const { reviewsVisible, reviews } = this.state;
+        if(reviewsVisible < reviews.length && reviews.length){
+            return (
+                <div class="block" style={{ textAlign: "center" }}>
+                    <button class="button is-primary" onClick={this.viewMore}>
+                        View more
+                    </button>
                 </div>
-              </div>
-            </div>
-
-          </div>
-        </section>
-       
-        <section id="Facilities" class="hero  is-light " style={{borderTop:"1px solid #ccd1d9"}} >
-          <div class="hero-body">
-
-            <h1 class="title" style={{ MarginTop: "0px" ,fontFamily: 'Pacifico' }}>
-              Aminent Facilities:
-            </h1>
-            <div class="content">
-              <div class="level-left">
-                <ul>
-                  <li><strong>Swimming pool</strong> </li>
-                  <li><strong>Basement Parking</strong></li>
-                  <li><strong>Gym</strong> </li>
-                </ul>
-              </div>
-            </div>
-
-
-          </div>
-        </section>
-        <section id="Location" class="hero  is-light " style={{borderTop:"1px solid #ccd1d9"}}>
-          <div class="hero-body">
-
-            <h1 class="title" style={{ MarginTop: "0px" ,fontFamily: 'Pacifico' }}>
-              Location:
-            </h1>
-            <div class='card-content is-flex' style={{ justifyContent: "center" }}>
-            <ReactMapGL {...this.state.viewport}
-              longitude={parseFloat(property.longitude)}
-              latitude={parseFloat(property.latitude)}
-              onViewportChange={(viewport) => this.setState({ viewport })}
-              mapStyle = "mapbox://styles/mapbox/streets-v11"
-              mapboxApiAccessToken = "pk.eyJ1IjoiZGtwMTkwMyIsImEiOiJjazhya3F3YWMwM2tsM21wbWY3ZTE0OWo1In0.bspO79N0Vc0Fgu1b4bqU5A"
-        >
-                <Marker
-                longitude={parseFloat(property.longitude)}
-                latitude={parseFloat(property.latitude)}
-              >
-                <span className="icon has-text-info">
-                  <i className="fa fa-home fa-lg" aria-hidden="true"></i>
-                </span>
-              </Marker>
-
-              </ReactMapGL>
-            </div>
-          </div>
-
-        </section>
+            );
+        }
+    }
+    
+    RenderWriteReview = () => {
+        if(localStorage.getItem("token")){
```

Comment:
this works but instead of using this connect to redux and access user from there, we need userId in this component